docs/running-aggregate-with-docker-compose.md

```diff
@@ -26,6 +26,8 @@ Run the following commands where you have copied your built setup. **Don't run t
 
 - Stop Aggregate by pressing `ctrl+c`
 
+[![asciicast](https://asciinema.org/a/190218.png)](https://asciinema.org/a/190218)
```

Comment:
I think asciicasts are cool, but this is the kinda thing that, IMHO, doesn't add a lot and is hard to maintain over time.

Reply:
I think it could relax anyone trying this at home to see how the expected outcome looks like but I guess you're right.